.cirrus.yml

```diff
@@ -1,10 +1,5 @@
-gcp_credentials: ENCRYPTED[!149d4005ecdba4cdd78bb5ba22756ebb98bf8e3367ee2e9ab08c5a1608c0d3e3b501904b67a1d67c0b63085e469d7dde!]
- 
 env:
   CIRRUS_CLONE_DEPTH: 20
-  CIRRUS_VAULT_AUTH_PATH: jwt-cirrusci
```

Comment:
why do you remove CIRRUS env here?
I haven't seen this as part of migration to AWS before ))

Reply:
It is provided here https://github.com/SonarSource/cirrus-modules/blob/master/features/vault.star#L14

Which is loaded by default.

Reply:
I see, thanks for the explanation. To me, this should be done as part of the migration to Vault.
David said that Evgeny did not want to use cirrus.star, which explains a lot.
Can you please double-check, if that's the case?

Reply:
> David said that Evgeny did not want to use cirrus.star

I never said this

Reply:
> > David said that Evgeny did not want to use cirrus.star
> 
> I never said this

Me neither, that was a guess